### PR TITLE
[PWGLF] Add new task for MFT Pb-Pb

### PR DESCRIPTION
**Pull request by gbencedi**

### CODE REVIEW

File: PWGMM/Mult/Tasks/dndeta-mft-pbpb.cxx [hunk 1]
```diff
@@ -0,0 +1,1504 @@
...
+#include "Common/DataModel/EventSelection.h"
+#include "Common/DataModel/Multiplicity.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "CommonConstants/MathConstants.h"
+
+#include "MathUtils/Utils.h"
+#include "ReconstructionDataFormats/GlobalTrackID.h"
+
+#include <TDatabasePDG.h>
+#include <TPDGCode.h>
```
Comment (vkucera) [inline PWGMM/Mult/Tasks/dndeta-mft-pbpb.cxx:43]: Include what you use.

**Comment by aalkin** [comment]
@gbencedi it is not properly communicated, but probably the easiest way to apply the required formatting is to accept the PR the format test workflow creates in your fork: https://github.com/gbencedi/O2Physics/pull/12

**Comment by vkucera** [comment]
> @gbencedi it is not properly communicated, but probably the easiest way to apply the required formatting is to accept the PR the format test workflow creates in your fork: [gbencedi#12](https://github.com/gbencedi/O2Physics/pull/12)

I think it is communicated very well. https://aliceo2group.github.io/analysis-framework/docs/gettingstarted/contributingtocode.html#automatic-checks

**Comment by gbencedi** [comment]
@aalkin @vkucera Indeed it is written well in the docs, only the notification about what to do was not straightforward for me at the beginning

File: PWGMM/Mult/Tasks/dndeta-mft-pbpb.cxx [hunk 2]
```diff
@@ -0,0 +1,1537 @@
...
+                                         tracks);
+  }
+
+  PROCESS_SWITCH(PseudorapidityDensityMFT, processMCInclusive,
+                 "Count MC particles", false);
+
+  /// @brief process fnc. to run on MC w FT0C centrality selection
+  void processMCCent(
+    filtMcGenColls::iterator const& mccollision,
+    soa::SmallGroups<soa::Join<filtCollsCent, aod::McCollisionLabels>> const& collisions,
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/dndeta-mft-pbpb.cxx:1094]: Note that having `Filtered` not as an outermost template (although inside `SmallGroups`) makes it ineffective. If you want the filters to be applied to collisions here, you need
`soa::SmallGroups<soa::Filtered<soa::Join<aod::Collisions, aod::EvSels, aod::McCollisionsLabels>>>>`

Comment (gbencedi) [reply PWGMM/Mult/Tasks/dndeta-mft-pbpb.cxx:1094]: Okay, I will fix that in another commit later on. Thanks for taking a careful look.

**Review by njacazio — APPROVED** [review]

**Pull request merged**